plugin/src/main/kotlin/tanvd/grazi/utils/JLangUtils.kt

```diff
@@ -69,4 +80,35 @@ object LangToolInstrumentation {
             }
         }
     }
+
+    fun reloadEnglish() {
+        val reader = ClassReader(Resources.getStream("/org/languagetool/language/English.class"))
+        val writer = ClassWriter(ClassWriter.COMPUTE_MAXS)
+        reader.accept(EnglishChunkerInstrumentation(writer), 0)
+
+        val cls = writer.toByteArray()
+        with(ClassLoader::class.java.getDeclaredMethod("defineClass", String::class.java, ByteArray::class.java, Int::class.java, Int::class.java)) {
```

Comment:
Huh, OK, found it here `https://asm.ow2.io/faq.html#Q5` :)

Reply:
Can we make this and others parts where we get methods via reflection extension functions and fields? For example define `ClassLoader.defineClass(...) = this code`

Reply:
Done